app/src/main/AndroidManifest.xml

```diff
@@ -163,11 +163,35 @@
         <action android:name="android.intent.action.MEDIA_BUTTON"/>
       </intent-filter>
     </service>
+    <service
+        android:name=".widgets.WidgetService"
```

Comment:
yeah - we can add Robolectric, but i am not sure how much you can test widgets with it - ideally i'd say let's just test any new "business logic" with just vanilla unit tests - since hopefully anything left is just the minimum required code for data / layout to the widget.

Reply:
I've updated/added tests in `BookmarkModelTest`.